### ISSUE TITLE
Can't open the build_installer.bat

### ISSUE THREAD

**Issue by xdark2050** [input-leap/input-leap#1881]
### What happened?

Hi, when i open it on windows to build it, it just closes.
i guess this is discontinued ;(
if you have any real kvm software to recommend me, please post it below

### Version

v2.4.0

### If applicable, where did you install Input Leap from?

Github

### What OSes are you seeing the problem on? (Check all that apply)

Windows

### What OS versions are you using?

11

**Comment by shymega** [comment]
Please read the wiki. You can download binaries from CI too.

Also, refrain from posts like this in future. This software is maintained, and it doesn't help to have comments about 'real KVM software.

Closing and locking.

**Comment by shymega** [comment]
Further info on why I locked and closed this issue:

- That BAT file is for use with the build system, not to be called on its own
- The wiki documents how to build, and we do provide Windows CI debug builds as well
- The comment about 'real KVM software', and asking for recommendations, _on an issue_, will lead to a flame war inevitably, and provide no helpful course for the project

I don't want the issue tracker to become a cesspool. That is why I closed and locked the issue. Input Leap is volunteer-led, we're not a paid product. And we intend to keep it that way.
